src/components/ActionsModal/types.ts

```diff
@@ -6,3 +6,8 @@ export interface ActionModalProps {
   onAddAction: (action: DecodedAction) => void;
   action?: DecodedAction;
 }
+
+export interface SelectedFunction {
+  functionName: string;

```

Comment:
To keep it consistent. Should this be either `functionName` and `functionTitle` or `name` and `title`?